pennylane/qaoa/mixers.py

```diff
@@ -112,3 +114,85 @@ def xy_mixer(graph):
         obs.append(qml.PauliY(node1) @ qml.PauliY(node2))
 
     return qml.Hamiltonian(coeffs, obs)
+
+
+def bit_flip_mixer(graph, n):
+    r"""Creates a bit-flip mixer Hamiltonian.
+
+    This mixer is defined as:
+
+    .. math:: H_M \ = \ \displaystyle\sum_{v \in V(G)} \frac{1}{2^d(v)} X_{v}
```

Comment:
```suggestion
    .. math:: H_M \ = \ \displaystyle\sum_{v \in V(G)} \frac{1}{2^{d(v)}} X_{v}
```
![image](https://user-images.githubusercontent.com/49409390/91729077-03c0fd80-eb72-11ea-9af2-a4b817d2ef37.png)